third_party/toolchains/BUILD

```diff
@@ -20,17 +20,17 @@ package(default_visibility = ["//visibility:public"])
 # Update every time when a new container is released.
 alias(
```

Comment:
I can still see reference to bazel_0.16.1 lower in this file
https://github.com/grpc/grpc/blob/8b7323df1c36bb855b749985ca301c5531d2e263/third_party/toolchains/BUILD#L85

Not sure what the comment at https://github.com/grpc/grpc/blob/8b7323df1c36bb855b749985ca301c5531d2e263/third_party/toolchains/BUILD#L77 means, but it seems that this was added by you in https://github.com/grpc/grpc/pull/17110, so you probably know.

Reply:
fixed